nebula/ui/components/forms/VPNTextArea.qml

```diff
@@ -11,6 +11,7 @@ import components 0.1
 
 Item {
     property alias placeholderText: formattedPlaceholderText.text
+    property alias textInput: textArea
```

Comment:
@flozia does this need to be `textArea.someProperty` ?

Reply:
@lesleyjanenorton This makes it possible to access all the properties and methods of `textInput`. That is why we can do `textInput.text`, `textInput.maxCharacterCount` and `textInput.forceActiveFocus()` from the tests for example.

Reply:
nice!